lib/internal/fs/streams.js

```diff
@@ -263,61 +242,37 @@ function WriteStream(path, options) {
     this.pos = this.start;
   }
 
+  Writable.call(this, options);
```

Comment:
This needs to be called after all values needed from `_construct` are initialized but before any Writable methods are needed.